docs/hardware/receiver-selection.md

```diff
@@ -13,14 +13,17 @@ ExpressLRS is hashtag blessed with the benefit of there being many receivers ava
 | Jumper AION Mini RX | :moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3: | :checkered_flag:|
 | Vantac RX | :moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3: | 20dBm | :whale::whale: | :thumbsup_tone3: | :checkered_flag:|
 | BetaFPV Nano 2.4 | :moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3:  | 20dBm | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
+| BetaFPV Lite 2.4 | :moneybag::moneybag: | :straight_ruler: | :thumbsup_tone3:  | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |
 | DIY Nano 2.4 | :moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale:/:whale::whale:/:whale::whale::metal: | :thumbsup_tone3:  | :thumbsdown_tone3: |
 | Flywoo EL24E | :moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |
 | Flywoo EL24P | :moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
 | Ghost ATTO | :moneybag::moneybag::moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsdown_tone3: | :thumbsdown_tone3: |
+| GEPRC Nano | :moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
 | HappyModel PP | :moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale: | :thumbsdown_tone3: | :checkered_flag: |
 | HappyModel EP1 | :moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
 | HappyModel EP2 | :moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |
 | HGLRC 2400RX | :moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: | 
+| iFlight ELRS | :moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
```

Comment:
It has LNA/PA equipped - 

| iFlight ELRS | :moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3: | 20dBm | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |

Reply:
I have updated this line.